src/actions/install-update.js

```diff
@@ -34,6 +34,7 @@ module.exports = function hydrator (params, callback) {
   let isJs = file.endsWith('package.json')
   let isPy = file.endsWith('requirements.txt')
   let isRb = file.endsWith('Gemfile')
+  let isDeno = file.endsWith('deps.ts') || file.endsWith('index.js') || file.endsWith('index.ts') || file.endsWith('index.tsx') || file.endsWith('mod.js') || file.endsWith('mod.ts') || file.endsWith('mod.tsx')
```

Comment:
I feel like we can tidy this up with an array + a `some()` check

Reply:
Yep, cool. No probs. Will push some amends later